### DISCUSSION TITLE
Codespace is unavailable

### DISCUSSION THREAD

**Discussion by nebulatgs** [community/community#4323] [Codespaces] [answered]
![image](https://user-images.githubusercontent.com/43934463/123429475-dbc8dd80-d57b-11eb-92de-e1b25179fa99.png)
I was working on my codespace when it disconnected and upon reconnecting, I was met with this screen. I have important uncommitted changes on this codespace as well.

**Comment by tanmayeekamath** [comment, marked as answer]
Hey @nebulatgs, apologies for a late response here, did a browser refresh help; are you still experiencing this issue?

**Reply by nebulatgs** [reply]
It resolved itself after an hour or two.

**Reply by tanmayeekamath** [reply]
Thanks for the update. Please reach out if this happens again.
